### PR TITLE
Feat/model relations

### PR DESCRIPTION
**Pull request by JWaltuch**
- Added relations for User, Trip, List Item, and Rating
- Updated seed to seed users, trips, list items, and ratings with appropriate relations

### CODE REVIEW

File: script/seed.js [hunk 1]
```diff
@@ -64,12 +65,83 @@ async function seed() {
...
+      {returning: true}
+    )
+  ])
+
   await trip1.addSubscriber(user2)
-  await trip2.setBuyer(user2)
   await trip2.addSubscriber(user1)
 
-  console.log(`seeded ${users.length} users, ${trips.length} trips`)
+  const listItems = await Promise.all([
```
Comment (katmd) [inline script/seed.js:89]: unitType is one of two options, 'single' or 'pack'.

File: script/seed.js [hunk 2]
```diff
@@ -64,12 +65,83 @@ async function seed() {
...
+        tripId: 1,
+        userId: 1,
+        type: 'buyer',
+        name: 'toilet paper',
+        price: 0.84,
+        qtyAvailable: 5,
+        qtyTotal: 5,
+        unitType: 'rolls',
+        isAccepted: 'approved'
+      },
```
Comment (katmd) [inline script/seed.js:101]: Missing amountDue: 4.2

File: script/seed.js [hunk 3]
```diff
@@ -64,12 +65,83 @@ async function seed() {
...
+        tripId: 2,
+        userId: 2,
+        type: 'buyer',
+        name: 'water bottle',
+        price: 0.84,
+        qtyAvailable: 5,
+        qtyTotal: 5,
+        unitType: 'bottles',
+        isAccepted: 'approved'
+      },
```
Comment (katmd) [inline script/seed.js:124]: Missing amountDue: 4.2

File: server/db/models/index.js [hunk 4]
```diff
@@ -1,23 +1,27 @@
 const User = require('./user')
-// const ListItem = require('./list-item')
+const ListItem = require('./list-item')
 const Trip = require('./trip')
-// const Rating = require('./rating')
+const Rating = require('./rating')
 
 User.belongsToMany(Trip, {as: 'subscriber', through: 'trip_subscribers'})
 Trip.belongsToMany(User, {as: 'subscriber', through: 'trip_subscribers'})
```
Comment (katmd) [inline server/db/models/index.js:7]: Suggest following camel case for TripSubscribers so potential requests to the join table are similar to those of other models

Comment (JWaltuch) [reply server/db/models/index.js:7]: Camel case makes it more difficult to use with psql. That's why I changed it. Is that ok?

Comment (katmd) [reply server/db/models/index.js:7]: Gotcha, keep it as is!

**Pull request merged**